lib/rspec/core/configuration.rb

```diff
@@ -141,7 +142,7 @@ def expose_dsl_globally=(value)
 
       # Determines where deprecation warnings are printed.
       # Defaults to `$stderr`.
-      # @return [IO, String] IO to write to or filename to write to
+      # @return [IO] IO to write to
```

Comment:
We moved stream preparation from `Formatters#Loader` to `Configuration`. And now `deprecation_stream` can’t contain a string here, because a deprecation stream already will be prepared and a string will be transformed to a file object.